tests/unit_tests/CMakeLists.txt

```diff
@@ -18,11 +18,21 @@
 #===============================================================================
 
 if("blas" IN_LIST TARGET_DOMAINS)
-  find_package(CBLAS REQUIRED)
+  find_package(CBLAS)
+  if(NOT CBLAS_FOUND)
+    # TODO: add list of tests without Netlib dependency
+    message(WARNING "Netlib CBLAS headers or libraries are not found, BLAS unit tests will be skipped")
+    list(REMOVE_ITEM TARGET_DOMAINS "blas")
```

Comment:
I would suggest to introduce a `TESTED_TARGET_DOMAINS` variable to avoid modifying `TARGET_DOMAINS` which is a global variable used elsewhere. This works fine as long as the tests are included last in the top-level CMakeLists.txt but I think it is best to avoid.